generate-spec

```diff
@@ -72,6 +72,9 @@ $infoXMLPath = $dir . "/appinfo/info.xml";
 
 if (file_exists($infoXMLPath)) {
 	$xml = simplexml_load_file($infoXMLPath);
```

Comment:
PHP EA (cool phpstorm plugin) als complains about this one:
![grafik](https://github.com/nextcloud/openapi-extractor/assets/213943/5c480f47-1691-4c73-a1ba-e8f7ff3bed46)


The result of "Replace with similar call" is:
```suggestion
	$xml = simplexml_load_string(file_get_contents($infoXMLPath));
```

We changed this in server as well at some point